spec/visual/samples/isbl

```diff
@@ -12,21 +7,21 @@
   ServiceFactory.GetUserByID(UserID)
   ExceptionsOn()
   if ExceptionExists()
-    Raise(CreateException("EDIRInvalidTypeOfParam";
-      LoadStringFmt("DIR21A2F148_1B41_40F3_9152_6E09E712025A"; "COMMON";
+    Raise(CreateException("EDIRInvalidTypeOfParam"; 

```

Comment:
By *trailing whitespace*, I meant these blank space after the semicolon..
There are many such instances in this commit..